core/configuration/ignored_identifier.py

```diff
@@ -1,57 +1,67 @@
-import json
 from typing import Optional, Sequence, Set, Union
 
 from flask_babel import lazy_gettext as _
 
-from core.config import ConfigurationTrait
-from core.model.configuration import (
-    ConfigurationAttributeType,
-    ConfigurationMetadata,
-    ConfigurationOption,
+from core.integration.settings import (
+    BaseSettings,
+    ConfigurationFormItem,
+    ConfigurationFormItemType,
+    FormField,
 )
 from core.model.constants import IdentifierType
+from core.model.integration import IntegrationConfiguration
+
+ALL_IGNORED_IDENTIFIER_TYPES = {
+    identifier_type.value for identifier_type in IdentifierType
+}
+
+
+class IgnoredIdentifierSettings(BaseSettings):
+    ignored_identifier_types: Optional[list] = FormField(
```

Comment:
I'd like to keep this change since lowercase is the norm and this is the only uppercase key.
The change will keep the keys somewhat standardized.

Reply:
I agree that this key should be lowercase, but since Pydantic is case sensitive, I don't understand why this can't just use an alias to do the case conversion. 

Is there some reason that this wouldn't work?

```python
    ignored_identifier_types: Optional[List[IdentifierType]] = FormField(
        default=[],
        alias="IGNORED_IDENTIFIER_TYPES",
        form=ConfigurationFormItem(
            label=_("List of identifiers that will be skipped"),
            description=_(
                "Circulation Manager will not be importing publications with identifiers having one of the selected types."
            ),
            type=ConfigurationFormItemType.MENU,
            required=False,
            options={
                identifier_type: identifier_type
                for identifier_type in ALL_IGNORED_IDENTIFIER_TYPES
            },
            format="narrow",
        ),
    )
```